packages/std/src/traits.rs

```diff
@@ -137,11 +135,48 @@ pub trait Querier: Clone + Send {
         self.query(&request)
     }
 
-    fn query_all_balances<U: Into<HumanAddr>>(&self, address: U) -> StdResult<AllBalanceResponse> {
+    fn query_all_balances<U: Into<HumanAddr>>(&self, address: U) -> StdResult<Vec<Coin>> {
         let request = BankQuery::AllBalances {
             address: address.into(),
         }
         .into();
-        self.query(&request)
+        let res: AllBalanceResponse = self.query(&request)?;
+        Ok(res.amount)
+    }
+
+    #[cfg(feature = "staking")]
+    fn query_validators(&self) -> StdResult<Vec<Validator>> {
+        let request = StakingQuery::Validators {}.into();
+        let res: ValidatorsResponse = self.query(&request)?;
+        Ok(res.validators)
+    }
+
+    #[cfg(feature = "staking")]
```

Comment:
You want to keep the feature conditions here? Just curious

Reply:
Yes, all helper functions are feature flagged, as a means to force contract writers to add the flag if they use it. (You ahd a good point not to remove it entirely).

I found this worked already with the hackatom contract, which did use one in a test case. So yeah, all data structures and things exported in `lib.rs` are now always available, it is a few little helpers that are staking specific that convince you to set the flag if needed (but without so intrusive flagging)